hack/e2e/kops/kops.sh

```diff
@@ -19,6 +19,9 @@
 
 set -euo pipefail
 
+# Allow deploying newer K8s clusters before kops releases catch up
+export KOPS_RUN_TOO_NEW_VERSION=true
```

Comment:
This file is not run directly, this should be done in `kops_create_cluster` (as well as the other functions if it is needed there)